app/data.ts

```diff
@@ -9,6 +9,7 @@ export const languages = [
   { value: "py", label: "Python" },
   { value: "rb", label: "Ruby" },
   { value: "rs", label: "Rust" },
+  { value: "ex", label: "Elixir" },
```

Comment:
Could you keep the list sorted alphabetically? So move Elixir so it's between CSS and Go.

Reply:
@jschr sure, let me adjust it